packages/store/src/utils.js

```diff
@@ -0,0 +1,43 @@
+// @ts-check
+
+const { getPrototypeOf, create, fromEntries } = Object;
+const { ownKeys, apply } = Reflect;
+
+const compareStringified = (left, right) => {
+  left = String(left);
+  right = String(right);
+  // eslint-disable-next-line no-nested-ternary
+  return left < right ? -1 : left > right ? 1 : 0;
+};
+
+/**
+ * @param {object} obj
+ * @returns {(string|symbol)[]}
+ */
+export const getMethodNames = obj => {
```

Comment:
these utils don't seem very specific to store.

I think we'd be well served by https://github.com/Agoric/agoric-sdk/issues/5992

Reply:
Hi @turadg ,
I just read #5992 . I like it a lot! But are you suggesting I should do this on the way to merging this PR?

In the past I have been flumoxed by what's needed to set up a new package. Just too much boilerplate that I'm unfamiliar with that I nevertheless need to get right. @kriskowal at one point had an automated shell script for automating new package creation for the endo repository, but I was never able to get it to work.

May I proceed to merge this PR without taking this step? Instead, I will put in a TODO into utils.js linking to #5992

Reply:
I added that TODO comment to `utils.js`

Reply:
SGTM. I don't see any reason for #5992 to block this. I'm mostly trying to drum up support for it. :)  I'd be happy to do the work if the issue is blessed. I'll assign it to myself.

Reply:
Thanks. I'll proceed with this PR now.

Yes, I like #5992 . Would love to see it happen -- thanks! Could you do it in such a way as to leave behind an understanding about all the boilerplate needed to start a new package?